pylatex/document.py

```diff
@@ -43,8 +46,26 @@ def __init__(self, default_filepath='default_filepath', *,
             than the standard LaTeX font.
         textcomp: bool
             Adds even more glyphs, for instance the Euro (€) sign.
+        lscape: bool
```

Comment:
Ah, I now see that they are now part of `geometry_options`. In that case, could you remove the documentation and the changelog information for the old ones.